### PR TITLE
Markdownlint

### PR DESCRIPTION
**Pull request by markstur**

### CODE REVIEW

File: .markdownlint.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            "*",
+            "## License"
+        ]
+    },
+    "spelling": false,
+    "no-alt-text": true,
+    "no-blanks-blockquote": false,
+    "html": {
+        "allowed_elements": [
+            "kbd"
```
Comment (stevemart) [inline .markdownlint.json]: why is kdb allowed?

Comment (markstur) [reply .markdownlint.json]: Originally copied from Dolph's.  Not sure what that is from.

File: .markdownlint.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
+    "_docs_for_this_file": "https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md",
+    "line-length": false,
+    "first-line-h1": true,
+    "first-header-h1": {
```
Comment (stevemart) [inline .markdownlint.json]: this is the default

File: .markdownlint.json (hunk 1 above)
Comment (markstur) [reply .markdownlint.json]: I'll remove it

File: .markdownlint.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        "level": 1
+    },
+    "header-style": {
+        "style": "atx"
+    },
+    "ul-style": {
+        "style": "asterisk"
+    },
+    "hr-style": {
```
Comment (stevemart) [inline .markdownlint.json]: i see line breaks used to infrequently that i'd just leave this out and go with the default "consistent"

File: .markdownlint.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+            "## Flow",
+            "## Included components",
+            "## Featured technologies",
+            "## Watch the Video",
+            "## Steps",
+            "*",
+            "## License"
+        ]
+    },
+    "spelling": false,
```
Comment (stevemart) [inline .markdownlint.json]: this isn't a thing? https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md

File: .markdownlint.json [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    "no-alt-text": true,
```
Comment (stevemart) [inline .markdownlint.json]: I dont see this one either

File: .markdownlint.json [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    "no-blanks-blockquote": false,
```
Comment (stevemart) [inline .markdownlint.json]: I would just use the default here

File: .markdownlint.json [hunk 7]
```diff
@@ -0,0 +1,33 @@
+{
+    "_docs_for_this_file": "https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md",
+    "line-length": false,
+    "first-line-h1": true,
```
Comment (stevemart) [inline .markdownlint.json]: i'd rather have this be false and not have to disable and enable things in the README...

File: .markdownlint.json (hunk 6 above)
Comment (markstur) [reply .markdownlint.json]: does not seem to have a default

File: .markdownlint.json (hunk 7 above)
Comment (markstur) [reply .markdownlint.json]: absolutely agree

**Comment by markstur** [comment]
I'm not very familiar with the defaults, but I tried removing most of the stuff and the JSON won't parse w/o those things.  We should get a better starting point for a reusable template, but actually this one is pretty close the minimum now.

**Pull request merged**